opendata_collector_module/metrics_opendata_collector/__init__.py

```diff
@@ -20,4 +20,4 @@
 #  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 #  THE SOFTWARE.
 
-__version__ = '0.1.0'
+__version__ = '1.2.0'
```

Comment:
@wisecrow is this correct that we're also bumping opendata collector's version?

Reply:
Since, opendata collector was never released before and this `__version_`is probably, dumb copy from other module, yes it has to be bumped.

Reply:
But I would ask Raido for confirmation also.

Reply:
Even if it hasn't been released before, let's have all the modules share the same version, so bumping this to `1.2.0` is good 👍